cmd/dicode/deno.go

```
@@ -0,0 +1,138 @@
+	if check {
+		cacheArgs = append(cacheArgs, "--frozen")
+	} else {
+		cacheArgs = append(cacheArgs, "--frozen=false")
+	}
+	cacheArgs = append(cacheArgs, "--lock="+lockPath)
+	if cfg := filepath.Join(dir, "deno.json"); fileExists(cfg) {
+		cacheArgs = append(cacheArgs, "--config="+cfg)
+	}
+	cacheArgs = append(cacheArgs, entrypoints...)
+
+	cmd := exec.Command(denoPath, cacheArgs...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)